src/tirith/providers/terraform_plan/handler.py

```diff
@@ -12,19 +16,18 @@ def _get_exp_attribute(split_expressions, input_data):
     # split_expressions=expression.split('*')
     final_data = []
     for i, expression in enumerate(split_expressions):
-        intermediate_val = pydash.get(input_data, expression)
-        # print(intermediate_val)
+        intermediate_val = pydash.get(input_data, expression, default=PydashPathNotFound)
         if isinstance(intermediate_val, list) and i < len(split_expressions) - 1:
             for val in intermediate_val:
                 final_attributes = _get_exp_attribute(split_expressions[1:], val)
                 for final_attribute in final_attributes:
                     final_data.append(final_attribute)
-        elif i == len(split_expressions) - 1 and intermediate_val:
+        elif i == len(split_expressions) - 1 and intermediate_val is not PydashPathNotFound:
             final_data.append(intermediate_val)
         elif ".*" in expression:
             intermediate_exp = expression.split(".*")
-            intermediate_data = pydash.get(input_data, intermediate_exp[0])
-            if intermediate_data and isinstance(intermediate_data, list):
+            intermediate_data = pydash.get(input_data, intermediate_exp[0], default=PydashPathNotFound)
+            if intermediate_data is not PydashPathNotFound and isinstance(intermediate_data, list):
```

Comment:
@refeed Can you please tell me the expected behavior in the following scenarios now?
1) Output when the value is found to be null
2) Output when the key is not found
3) How is the core handling both the cases ?

Reply:
Sure

1. Output when the value is found to be null -> this function will return `[None]` and will send it to core, then will be handled by the evaluator
2. Output when the key isn't found -> this function will return nothing or `None` to the caller (implicitly), then will be send to core, then will be skipped by core (in the current implementation)